### ISSUE TITLE
RLbench dataset

### ISSUE THREAD

**Issue by fengxiuyaun** [nickgkan/3d_diffuser_actor#12]
Hi,
Thanks for your good work!
I visualize the RLBench training set (https://huggingface.co/katefgroup/3d_diffuser_actor/blob/main/Peract_packaged.zip) and find that there is a significant difference from the peract raw(https://drive.google.com/drive/folders/0B2LlLwoO3nfZfkFqMEhXWkxBdjJNNndGYl9uUDQwS1pfNkNHSzFDNGwzd1NnTmlpZXR1bVE?resourcekey=0-jRw5RaXEYRLe2W6aNrNFEQ), as shown in the figure.
<img width="541" alt="image" src="https://github.com/nickgkan/3d_diffuser_actor/assets/6481257/f9da5592-d17e-4ae2-97ac-ffb1a723920b">
<img width="262" alt="image" src="https://github.com/nickgkan/3d_diffuser_actor/assets/6481257/2b851683-18a9-44d9-a97b-1ae5a8f72a7a">

Which one is OK?

**Comment by twke18** [comment]
Hi, the figures you showed are from different tasks.  Would you like to double check?  Our packaged training data is re-rendered with the same initial object states as those provided from Peract.  The only difference is in the target gripper poses, as the motion planner used to generate the expert demonstrations is still stochastic, even by setting the same random seed state.

**Comment by nickgkan** [comment]
Hi, I suppose you mean texture difference. I suspect the images you loaded from our packaged data have values in the [-1, 1] range. If this is the case, your visualization code needs to be adjusted a bit, so that you map our images to the correct range (try [0, 1] float or [0, 255] uint8).

Other than that, what @twke18 is correct, you should be able to see the same scenes for the same tasks.

**Comment by fengxiuyaun** [comment]
Oh, sorry. I understand now.

> Hi, I suppose you mean texture difference. I suspect the images you loaded from our packaged data have values in the [-1, 1] range. If this is the case, your visualization code needs to be adjusted a bit, so that you map our images to the correct range (try [0, 1] float or [0, 255] uint8).
> 
> Other than that, what @twke18 wrote is correct, you should be able to see the same scenes for the same tasks.
